README.md

```diff
@@ -30,14 +30,37 @@ Full documentation [is available here](http://docs.docker.com/swarm/).
 You can download and install from source instead of using the Docker
 image. Ensure you have golang, godep and the git client installed.
 
+Note: Starting from Swarm 0.4 golang 1.4.x or later is required for building, so
```

Comment:
@moxiegirl Thanks a lot for your comments. I am agree with you to remove the details to install each prerequisite software.